src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -400,9 +399,7 @@ def setupTask(self, task, taskConf):
                                               splitArgs = taskConf["SplittingArguments"],
                                               stepType = cmsswStepType,
                                               forceUnmerged = forceUnmerged,
-                                              timePerEvent = self.timePerEvent,
                                               memoryReq = taskConf.get("Memory", None),
-                                              sizePerEvent = self.sizePerEvent, 
```

Comment:
timePerEvent and sizePerEvent needs to passed in this call. The same applies to the call from setupGeneratorTask()